### PR TITLE
Init Apollo client

### PR DESCRIPTION
**Pull request by svirgunvolodia**
issue: https://github.com/WiseEngineering/crispy-web/issues/5
needs: https://github.com/WiseEngineering/crispy-api/issues/4

### CODE REVIEW

File: src/utils/apollo/withQuery.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+          error={error}
+          data={data}
+          refetch={refetch}
+          fetchMore={fetchMore}
+        />
+      )
+    }
+
+    return (
+      <Query
```
Comment (melnychukvitaliy) [inline src/utils/apollo/withQuery.tsx]: we have to use our `Query` for start and then reuse that `Query` in `withQuery` HOC. 

In this case, let's create `components/Query` with logic as here and then reuse it in this HOC

File: src/ApolloProvider.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  link: ApolloLink.from([
+    onError(({ graphQLErrors, networkError }) => {
+      if (graphQLErrors)
+        graphQLErrors.map(({ message, locations, path }) =>
+          console.log(`[GraphQL error]: Message: ${message}, Location: ${locations}, Path: ${path}`)
+        )
+      if (networkError) console.log(`[Network error]: ${networkError}`)
+    }),
+    new HttpLink({
+      uri: '/crispy-api',
```
Comment (melnychukvitaliy) [inline src/ApolloProvider.tsx]: have you tried how it will work with API?
I think, `api` name would be better

Comment (svirgunvolodia) [reply src/ApolloProvider.tsx]: I will change all URL's once the crispy-api endpoint would be ready

File: src/utils/apollo/withQuery.tsx (hunk 1 above)
Comment (svirgunvolodia) [reply src/utils/apollo/withQuery.tsx]: :ok_hand:

File: src/ApolloProvider.tsx (hunk 2 above)
Comment (Bazillius) [reply src/ApolloProvider.tsx]: I guess we should move it to config, and get `uri` from config

File: codegen.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+overwrite: true
+schema: "https://crispy-api/graphql"
```
Comment (melnychukvitaliy) [inline codegen.yml]: is this url correct?
@Bazillius do we need nginx over `web`and `api` to proxy requests?

Comment (svirgunvolodia) [reply codegen.yml]: This URL would be also changes when we will know exactly which is correct

**Pull request merged**